### PR TITLE
DOC fixing docstrings

### PR DESCRIPTION
**Pull request by aabadie**
Other fixes with docstrings.

### CODE REVIEW

**Comment by aabadie** [comment]
ping  @lesteve. Not interested with this one ?

**Comment by lesteve** [comment]
> ping  @lesteve. Not interested with this one ?

I don't feel the passion I have to say :-) ! Do you have any reference that requires the first line of a docstring to end with a dot? Personally I wouldn't bother especially for one-line docstrings.

File: joblib/pool.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 This module should not be imported if multiprocessing is not
 available as it implements subclasses of multiprocessing Pool
-that uses a custom alternative to SimpleQueue.
+which uses a custom alternative to SimpleQueue.
```
Comment (lesteve) [inline joblib/pool.py]: I don't think we should start splitting hair about that vs which. You can google it though and potentially learn something (I just did).

File: joblib/pool.py [hunk 2]
```diff
@@ -315,15 +316,15 @@ class CustomizablePicklingQueue(object):
 
     This class is an alternative to the multiprocessing implementation
     of SimpleQueue in order to make it possible to pass custom
-    pickling reducers, for instance to avoid memory copy when passing
+    pickling reducers. For instance, it avoids memory copy when passing
```
Comment (lesteve) [inline joblib/pool.py]: The original sentence seems fine to me.

File: joblib/pool.py [hunk 3]
```diff
@@ -315,15 +316,15 @@ class CustomizablePicklingQueue(object):
...
 
-    `reducers` is expected expected to be a dictionary with key/values
-    being `(type, callable)` pairs where `callable` is a function that
-    give an instance of `type` will return a tuple `(constructor,
-    tuple_of_objects)` to rebuild an instance out of the pickled
-    `tuple_of_objects` as would return a `__reduce__` method. See the
-    standard library documentation on pickling for more details.
+    `reducers` is expected to be a dictionary with key/values
+    being `(type, callable)` pairs where `type` is an instance and `callable`
+    is a function that returns a tuple `(constructor, tuple_of_objects)`
```
Comment (lesteve) [inline joblib/pool.py]: I am wondering whether the original sentence was not supposed to be "that given a function of ...", i.e. the callable takes `type` as input argument and returns a tuple.

File: joblib/pool.py [hunk 4]
```diff
@@ -315,15 +316,15 @@ class CustomizablePicklingQueue(object):
...
+    to rebuild an instance out of the pickled `tuple_of_objects` as would
+    return a `__reduce__` method.
+    See the standard library documentation on pickling for more details.
```
Comment (lesteve) [inline joblib/pool.py]: Was the line above too long ?

File: joblib/pool.py [hunk 5]
```diff
@@ -437,7 +438,7 @@ def delete_folder(folder_path):
 
 
 class MemmapingPool(PicklingPool):
-    """Process pool that shares large arrays to avoid memory copy.
+    """Process pool sharing large arrays to avoid memory copy.
```
Comment (lesteve) [inline joblib/pool.py]: The original sentence seemed fine

File: joblib/pool.py [hunk 6]
```diff
@@ -1,4 +1,4 @@
-"""Custom implementation of multiprocessing.Pool with custom pickler
+"""Custom implementation of multiprocessing.Pool with custom pickler.
```
Comment (aabadie) [inline joblib/pool.py]: Just for the record, the period seems required by pep257. https://www.python.org/dev/peps/pep-0257/#one-line-docstrings

File: joblib/pool.py (hunk 4 above)
Comment (aabadie) [inline joblib/pool.py]: Nope, it was just a cosmit change.

File: joblib/pool.py (hunk 5 above)
Comment (aabadie) [inline joblib/pool.py]: Ok, will revert this one.

File: joblib/pool.py (hunk 6 above)
Comment (lesteve) [inline joblib/pool.py]: Fair enough, let's leave it like that then.

**Comment by lesteve** [comment]
LGTM, merging, thanks!

**Pull request merged**